packaging/linux/common/notes.appdata.xml.in

```diff
@@ -56,7 +56,18 @@
     </screenshots>
     <content_rating type="oars-1.1" />
     <releases>
-    <release date="2024-08-31" version="2.3.0">
+        <release date="2024-09-30" version="2.3.1">
+            <description>
```

Comment:
@nuttyartist Please let me know if this change log is written to your liking.

Reply:
Looks good to me.